src/payment/checkout/payment-form/flex-microform/FlexMicroform.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { setPaymentTokenFetch } from '../../../payment-methods/cybersource/service';
+
+class FlexMicroform extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            microform: null,
+        }
+        this.onLoad = this.onLoad.bind(this);
+        this.getPaymentToken = this.getPaymentToken.bind(this);
+    }
+
+    componentDidMount() {
+        setPaymentTokenFetch(this.getPaymentToken);
+
+        const script = document.createElement('script');
+        script.src = "https://flex.cybersource.com/cybersource/assets/microform/0.11/flex-microform.min.js";
+        script.async = true;
+
+        script.addEventListener('load', this.onLoad);
+        script.addEventListener('error', e => {
+            console.error(e);
+        });
+
+        document.body.appendChild(script);
+    }
+
+    initialize() {
+        if (this.state.microform !== null) {
+            return;
+        }
+        if (!this.props.captureKeyId) {
+            return;
+        }
+        if (window['Flex'] === undefined) {
+            return;
+        }
+        const microform = new window.Flex(this.props.captureKeyId).microform({
+            styles: { 
+                'input': {    
+                    'font-size': '16px',
+                    'font-family': 'Roboto, "Helvetica Neue", Arial, sans-serif',
+                    'color': '#2d323e',
+                },
+                ':disabled': {
+                    'cursor': 'not-allowed'
+                }
+            }
+        });
+        this.setState({microform: microform});
```

Comment:
Note that while it's not a requirement, state should generally be serializable data.  The concern is around side effects/internal state changes that may occur in the object not being 'noticed' by React if they don't take place through calls to `setState` or changes in props.  If a piece of data in an object is changed outside React, it won't know the data changed and won't be able to re-render as a result, which can cause undefined behavior.

Reply:
You may consider setting microform as an instance variable on the component, which would indicate it's not supposed to be considered when re-rendering, but would let you maintain a handle to it.